### PR TITLE
Updates to real-time model using first batch of 2026 samples

### PR DESCRIPTION
**Pull request by SamStruthers**
Hi @steeleb, 
Here is my script for the updated model version (+ some other bugs I caught along the way). Main thing to check out is `modeling/toc/real_time/toc_model_20260715.Rmd`. Most of this script will mirror previous versions of the scripts in the `modeling/realtime/toc` folder but I have simplified things where possible. The knitted version is in `docs/uclp_dss/docs/knit_markdowns/realtime_modeling/toc/toc_model_20260715.html`. 

Overall performance is still about the same and we still struggle on the highest TOC values BUT I think this set of models appears to be more consistent across folds. Still opted to not use lagged parameters or sin(doy) since we still are really only working with 2 years of data and I am concerned that we would likely overfit to a "normal" TOC year 

@danny-duncan Just adding you here for reference so you are aware of how this was assembled!

### CODE REVIEW

**Comment by steeleb** [comment]
Hey Sam!!

This looks great. Glad to see some marginal improvements in the model, too. 

The only thing I'd flag is in Fold 1 it looks like we're possibly hitting a training wall of some sort (all of the validation loss curves increase after the minima), and a littttttle bit in Fold 4. These are also the folds that happen to have the worst low-end benching. Mostly an observation. I think the only way to figure out what's going on here is to manually play with hyperparameters to see if we are just over/underparameterized somewhere. 

I don't need any changes here - good to merge.

**Review by steeleb — APPROVED** [review]

**Comment by SamStruthers** [comment]
Thanks for checking this out!
